src/widgets/Landing.tsx

```diff
@@ -0,0 +1,10 @@
+import WidgetBase from '@dojo/framework/widget-core/WidgetBase';
+import { tsx } from '@dojo/framework/widget-core/tsx';
+import { WidgetProperties } from '@dojo/framework/widget-core/interfaces';
+import * as css from './Landing.m.css';
+
+export default class Landing extends WidgetBase<WidgetProperties> {
```

Comment:
A standalone `Landing` widget was created so this styling (and potential functionality in the future) could be used outside of the content pipeline.